packages/components/app/styles/components/app-footer.scss

```diff
@@ -0,0 +1,162 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// app-footer
+//
+
+// WIP: names, etc. may change
+// :root {
+//   --token-app-footer-text-color: var(--token-color-foreground-faint);
+//   --token-app-footer-border-color: var(--token-color-border-primary);
+// }
+
+$app-footer-gap: 24px;
+
+.hds-app-footer {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $app-footer-gap;
+  justify-content: flex-end;
+  padding: 24px;
+  border-top: 1px solid;
+}
+
+.hds-app-footer__list,
+.hds-app-footer__legal-links {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $app-footer-gap;
+  align-items: center;
+  justify-content: flex-end;
+  width: max-content;
+  max-width: 100%;
+  margin: 0;
+  padding: 0;
+}
+
+.hds-app-footer__list-item { list-style-type: none; }
+
+.hds-app-footer__status-link {
+  .flight-icon {
+    fill: var(--hds-app-footer-status-icon-color, currentColor);
+  }
+}
+
+.hds-app-footer__status-link,
+.hds-app-footer__list-item,
+.hds-app-footer__copyright {
+  display: flex;
+  gap: 6px;
+  align-items: center;
+  justify-content: center;
+
+  // temporary to fix layout bug
+  .hds-link-inline--icon-leading > .hds-link-inline__icon { margin: 0; }
+}
+
+// Themes
+// ------------
+
+// Light
+// Q: Should the light theme colors be applied by default instead of class-based?
```

Comment:
Let's discuss the approach here, I think we should base the whole theming on CSS variables, so it's easy to describe in code (and to maintain, also in the future), plus easy to overwrite from the consumers if they need something specific for a special use case (eg an internal app).